easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock='Binary'
+
+name = "T-Coffee"
```

Comment:
They should be, but Travis gave the green light on this PR before the style checks were enabled.